apps/emqx_plugin_libs/src/emqx_placeholder.erl

```diff
@@ -28,7 +28,7 @@
     preproc_sql/1,
     preproc_sql/2,
     proc_sql/2,
-    proc_sql_param_str/2,
+    proc_mysql_param_str/2,
```

Comment:
I think adding a new function with this name may be better than replacing the generic function, there are many SQL-likes backend.

Reply:
Did that as part of https://github.com/emqx/emqx/pull/10095/commits/2544742b0cf02aed1707b068237bb3762651b663. I tried to raise some awareness there about `pgsql` in a comment, would be glad if you take another look.